pkg/auth/iam/iam.go

```diff
@@ -62,6 +65,8 @@ type ErrorResponse struct {
 
 // NewFilter creates new Filter instance
 func NewFilter(client iam.Client) *Filter {
+	SetDevStackTracer()
```

Comment:
another thinking why do we need to add the function call in multiple methods explicitly? Can we reach it by init()

@marselsampe, any ideas?

Reply:
sounds good, better to use `init()` for this use case

Reply:
I just realized the power of `init()`. I will changes the `SetDevStackTracer()` to just use `init()` 
@angelo-chan @marselsampe